site/app/controllers/forum/ForumController.php

```diff
@@ -171,8 +171,9 @@ private function isCategoryDeletionGood($category_id) {
      */
     public function addNewCategory($category = []) {
         $result = [];
-        if (!empty($_POST["newCategory"])) {
+        if (!empty($_POST["newCategory"]) and !empty($_POST["visibleDate"])) {
             $category = trim($_POST["newCategory"]);
+            $visibleDate = trim($_POST["visibleDate"]);
```

Comment:
Should probably add some sort of validation here. Could use DateTime

Reply:
This still needs some sort of validation that it actually is a date and not some random string or other value